### DISCUSSION TITLE
How setup custom highlight  from third party plugin?

### DISCUSSION THREAD

**Discussion by KirovAI** [NvChad/NvChad#1350] [Q&A] [answered]
I install this plugin https://github.com/yamatsum/nvim-cursorline , using next settings:
![cursor_line_config](https://user-images.githubusercontent.com/105173528/179507725-0d4e7859-382d-4f30-a5a5-f096a4cad759.png)
![cursor_line_config](https://user-images.githubusercontent.com/105173528/179507965-cacf9b2a-2c1f-4ddd-b210-358eb19f3477.png)
And after installation plugin dont work. No one word higlighted. What should I do?
Thanks in advance

**Comment by siduck** [comment, marked as answer]
@KirovAI  seems like this plugin defines 'CursorWord' highight group 

https://github.com/yamatsum/nvim-cursorline/blob/main/lua/nvim-cursorline.lua#L90

so add that highlight group https://nvchad.github.io/config/theming#add-new-highlight-groups

```lua
["yamatsum/nvim-cursorline"] = {
      event = "CursorMoved",
      config = function()
         require("nvim-cursorline").setup {}
      end,
   },
   ```
   
   Edit : added the answer!

**Reply by KirovAI** [reply]
its partially solved problem cuz after :PackerSync its worked(but without underline, sadly that base 46 not provide it) ,  after exit and entire in neovim I again see that highlight not work! You wouldnt advice something for solving? 
config of chadrc:

![cursor_line_config](https://user-images.githubusercontent.com/105173528/179556613-c11c5e38-66fa-480b-a4fa-5f1bf9e35ffd.png)

and above config of "neovim-cursorline" plugin
P.S.: sorry, I very stupid and understand'nt many things. It cause all my question(

**Reply by siduck** [reply]
does it work now?

**Comment by KirovAI** [comment]
No. If I not use :PackerSync in even entry in neovim it dosnt  work

**Reply by siduck** [reply]
try loading nvimcursorline after base46 plugin

**Reply by siduck** [reply]
@KirovAI got a fix, try lazyloading it 

```lua
["yamatsum/nvim-cursorline"] = {
      event = "CursorMoved",
      config = function()
         require("nvim-cursorline").setup {}
      end,
   },
   ```
   do packercompile

**Reply by KirovAI** [reply]
Hooray!!! thank you , all working
